instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetServerAttributesExtractorTest.java

```diff
@@ -25,22 +25,34 @@
 class NetServerAttributesExtractorTest {
 
   static class TestNetServerAttributesGetter
-      implements NetServerAttributesGetter<Map<String, String>> {
+      implements NetServerAttributesGetter<Map<String, String>, Void> {
 
     @Override
     public String getTransport(Map<String, String> request) {
+      return request.get("netTransport");
+    }
+
+    @Nullable
+    @Override
+    public String getNetworkTransport(Map<String, String> request, @Nullable Void response) {
       return request.get("transport");
```

Comment:
It probably doesn't matter too much, but maybe we could use the real names in the test to help prevent a reader from thinking these are the actual names. Maybe use the real constants. It it requires a bunch of changes that of course could be done in a follow-up PR.

```suggestion
      return request.get("net.transport");
```